Formula/thrift.rb

```diff
@@ -1,8 +1,8 @@
 class Thrift < Formula
   desc "Framework for scalable cross-language services development"
   homepage "https://thrift.apache.org/"
-  url "https://www.apache.org/dyn/closer.cgi?path=/thrift/0.11.0/thrift-0.11.0.tar.gz"
-  sha256 "c4ad38b6cb4a3498310d405a91fef37b9a8e79a50cd0968148ee2524d2fa60c2"
+  url "https://www.apache.org/dyn/closer.cgi?path=/thrift/0.12.0/thrift-0.12.0.tar.gz"
```

Comment:
This url isn't working for me:
```
MacBook-Pro-van-Fokko:parquet-mr fokkodriesprong$ curl -o thrift-0.12.0.tar.gz -L  https://www.apache.org/dyn/closer.cgi?path=/thrift/0.12.0/thrift-0.12.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 18711    0 18711    0     0  77732      0 --:--:-- --:--:-- --:--:-- 77639
MacBook-Pro-van-Fokko:parquet-mr fokkodriesprong$ ls -lah thrift-0.12.0.tar.gz 
-rw-r--r--  1 fokkodriesprong  staff    18K 24 jan 20:01 thrift-0.12.0.tar.gz
MacBook-Pro-van-Fokko:parquet-mr fokkodriesprong$ head thrift-0.12.0.tar.gz 
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Home page of The Apache Software Foundation">
  <base href="https://www.apache.org/dyn/closer.cgi" />
  <link rel="apple-touch-icon" sizes="57x57" href="/favicons/apple-touch-icon-57x57.png">
  <link rel="apple-touch-icon" sizes="60x60" href="/favicons/apple-touch-icon-60x60.png">
```

Reply:
I think it depends on the user agent and other settings, it's used everywhere throughout homebrew.

Reply:
Yeah, this link redirects to the closest mirror